src/core/components/AMInstallButton/index.js

```diff
@@ -115,6 +115,10 @@ export class AMInstallButtonBase extends React.Component<InternalProps> {
       addonName: addon.name,
       type: addon.type,
     });
+    this.trackInstallSucceeded({
+      addonName: addon.name,
+      type: addon.type,
```

Comment:
since type and name are used a few times, maybe we can extract them out here?

Reply:
yup